src/assets/css/components/button.css

```diff
@@ -0,0 +1,22 @@
+.btn {
+    color: var(--primary-color);
+    font-family: var(--primary-font);
+    font-size: 16px;
+    font-weight: var(--fw-semibold);
+  }
+  .btn--secondary {
+    padding: 10px 22.22px 9px 25.25px;
+    margin-right: 32px;
+    text-decoration: none;
+    flex-shrink: 0;
+    border-radius: 4px;
+    border: 1px solid #bcd0e5;
+    text-align: center;
+  }
+  .btn--primary {
+    padding: 10px 23px 9px 25px;
+    text-decoration: none;
+    flex-shrink: 0;
+    border-radius: 4px;
+    background: #02897a;
+  }
```

Comment:
With the button, should split into another PR

Reply:
Thank you, I will create a new branch and PR it